docs/docsite/rst/scenario_guides/guide_meraki.rst

```diff
@@ -0,0 +1,75 @@
+.. _meraki_guide:
+
+Cisco Meraki Guide
+==================
+
+
+.. _meraki_guide_intro:
+
+What is Cisco Meraki?
+---------------------
+
+Cisco Meraki is an easy to use, cloud based, network infrastructure platform for enterprise environments. While most network hardware uses command line interfaces (CLIs) for configuration, Meraki uses an easy to use Dashboard hosted in the Meraki cloud. No on-premises management hardware or software is required - only the network infrastructure to run your business.
+
+MS Switches
+...........
+
+Meraki MS switches come in multiple flavors and form factors. Meraki switches support 10/100/1000/10000 ports, as well as Cisco's mGig technology for 2.5/5/10Gbps copper connectivity. 8, 24, and 48 port flavors are available with PoE (802.3af/802.3at/UPoE) available on many models.
+
+MX Firewalls
+............
+
+Meraki's MX firewalls support full layer 3-7 deep packet inspection. MX firewalls are compatible with a variety of VPN technologies including IPSec, SSL VPN, and Meraki's easy-to-use AutoVPN.
+
+MR Wireless Access Points
+.........................
+
+MR access points are enterprise class, high performance, access points for the enterprise. MR access points have MIMO technology and integrated beamforming built-in for high performance applications. BLE allows for advanced location applications to be developed with no on-premises analytics platforms.
+
+Using the Meraki modules
+------------------------
+
+Meraki modules provide a user-friendly interface to manage your Meraki environment using Ansible. For example, details about SNMP settings for a particular organization can be discovered using the module `meraki_snmp <meraki_snmp_module>`.
+
+.. code_block:: yaml
```

Comment:
Try this maybe ?
```
::
    - name: Query SNMP settings
      meraki_snmp:
          api_key: abc123
          org_name: AcmeCorp
          state: query
          delegate_to: localhost
```
I couldn't find any instance of ``code_block`` by grepping the docs.

Reply:
Unscores and dashes are important.